src/domain/harbors/HarborsPageContainer.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import { useTranslation } from 'react-i18next';
+
+import { HARBORS_QUERY } from './harborsQuery';
+import { HARBORS } from './__generated__/HARBORS';
+import PropertyIcon from './propertyIcon/PropertyIcon';
+import Table from '../../common/table/Table';
+
+const HarborsContainer: React.FC = () => {
+  const { loading, error, data } = useQuery<HARBORS>(HARBORS_QUERY);
+  const { t } = useTranslation();
+
+  const columns = [
+    {
+      Header: t('harbors.tableHeaders.harbor'),
+      accessor: 'name',
+    },
+    {
+      Header: t('harbors.tableHeaders.places'),
+      accessor: 'numberOfPlaces',
+    },
+    {
+      Cell: ({ cell }: { cell: { value: number } }) => (
+        <PropertyIcon name="plug" disabled={!!cell.value} />
+      ),
+      Header: () => <PropertyIcon name="plug" />,
+      accessor: 'electricity',
+    },
+    {
+      Cell: ({ cell }: { cell: { value: number } }) => (
+        <PropertyIcon name="fence" disabled={!!cell.value} />
+      ),
+      Header: () => <PropertyIcon name="fence" />,
+      accessor: 'gate',
+    },
+    {
+      Cell: ({ cell }: { cell: { value: number } }) => (
+        <PropertyIcon name="streetLight" disabled={!!cell.value} />
+      ),
+      Header: () => <PropertyIcon name="streetLight" />,
+      accessor: 'lighting',
+    },
+    {
+      Cell: ({ cell }: { cell: { value: number } }) => (
+        <PropertyIcon name="waterTap" disabled={!!cell.value} />
+      ),
+      Header: () => <PropertyIcon name="waterTap" />,
+      accessor: 'water',
+    },
+    {
+      Cell: ({ cell }: { cell: { value: number } }) => (
```

Comment:
Those thing are repeated, can you export that to something shorter ? function ? array.map rendering?

Reply:
Yes, this array wasn't typed, so I added the type, which allowed me to get rid of these repeated inline types `{ cell: { value: number } }`.
But other than that and after trying a couple of ways, I figured that things would only get more complicated since I still have to list all the values above.
Let me know if you have another solution!